src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/cache.html.twig

```diff
@@ -54,93 +54,115 @@
         </div>
         <div class="metric">
             <span class="value">{{ collector.totals.deletes }}</span>
             <span class="label">Total deletes</span>
         </div>
+        <div class="metric-divider"></div>
         <div class="metric">
-            <span class="value">{{ collector.totals['hits/reads'] }}</span>
+            <span class="value">{{ collector.totals.hits }}</span>
+            <span class="label">Total hits</span>
+        </div>
+        <div class="metric">
+            <span class="value">{{ collector.totals.misses }}</span>
+            <span class="label">Total misses</span>
+        </div>
+        <div class="metric">
+            <span class="value">
+                {% if collector.totals.hit_read_ratio is null %}
+                    n/a
+                {% else %}
+                    {{ collector.totals.hit_read_ratio }} <span class="unit">%</span>
+                {% endif %}
+            </span>
             <span class="label">Hits/reads</span>
         </div>
     </div>
 
-    {% for name, calls in collector.calls %}
-        <h3>Statistics for '{{ name }}'</h3>
-        <div class="metrics">
-            {% for key, value in collector.statistics[name] %}
-                <div class="metric">
-                    <span class="value">
-                        {% if key == 'time' %}
-                            {{ '%0.2f'|format(1000*value.value) }} ms
-                        {% else %}
-                            {{ value }}
-                        {% endif %}
-                    </span>
-                    <span class="label">{{ key|capitalize }}</span>
-                </div>
-            {% endfor %}
-        </div>
-        <h4>Calls for '{{ name }}'</h4>
+    <div class="sf-tabs">
+        {% for name, calls in collector.calls %}
+            <div class="tab {{ calls|length == 0 ? 'disabled' }}">
+                <h3 class="tab-title">{{ name }} <span class="badge">{{ collector.statistics[name].calls }}</span></h3>
 
-        {% if not collector.totals.calls %}
-            <p>
-                <em>No calls.</em>
-            </p>
-        {% else %}
-            <table>
-                <thead>
-                <tr>
-                    <th style="width: 5rem;">Key</th>
-                    <th>Value</th>
-                </tr>
-                </thead>
-                <tbody>
-                {% for i, call in calls %}
-                    <tr>
-                        <th style="padding-top:2rem">#{{ i }}</th>
-                        <th style="padding-top:2rem">Pool::{{ call.name }}</th>
-                    </tr>
-                    <tr>
-                        <th>Argument</th>
-                        <td>{{ profiler_dump(call.argument, maxDepth=2) }}</td>
-                    </tr>
-                    <tr>
-                        <th>Results</th>
-                        <td>
-                            {% if call.result != false %}
```

Comment:
@Nyholm not sure about a) this check and b) loose comparison. As you can see in the screenshots `null` was hidden as wel. Does `false` need to be excluded, strictly?